pkg/importer/importer.go

```diff
@@ -15,19 +15,20 @@ import (
 type State interface {
 	AcceptAndVerifyBlockBinary(block []byte, initialisation bool) error
 	GetBlockByHeight(height uint64) (*proto.Block, error)
-	WavesAddressesNumber() (uint64, error)
+	AddressesNumber() (uint64, error)
 	AccountBalance(addr proto.Address, asset []byte) (uint64, error)
 }
 
-func ApplyFromFile(st State, blockchainPath string, nBlocks int, checkBlocks bool) error {
+func ApplyFromFile(st State, blockchainPath string, nBlocks, startHeight uint64, checkBlocks bool) error {
 	blockchain, err := os.Open(blockchainPath)
 	if err != nil {
 		return errors.Errorf("failed to open blockchain file: %v\n", err)
 	}
 	sb := make([]byte, 4)
 	buf := make([]byte, 2*1024*1024)
```

Comment:
Better create not slice of bytes, but fixed byte array of the maximum block size. And later create a slice of required length from it.

Reply:
Done.